packages/ceramic-core/src/doctypes/accountLinkHandler.ts

```diff
@@ -0,0 +1,74 @@
+import DoctypeHandler from './doctypeHandler'
+import { DocState, SignatureStatus, AnchorProof } from '../document'
+import { validateLink } from '3id-blockchain-utils'
+
+const DOCTYPE = 'account-link'
+
+function head (log: Array<string>): string {
+  return log[log.length - 1]
+}
+
+class AccountLinkHandler extends DoctypeHandler {
+  constructor () {
+    super(DOCTYPE)
+  }
+
+  async makeRecord (state: DocState, proof: any): Promise<any> {
+    const record = { content: proof, next: { '/': head(state.log) } }
+    return record
+  }
+
+  async makeGenesis (content: any, owners: [string]): Promise<any> {
+    if (content) throw new Error('Account link genesis cannot have content')
+    if (!owners) throw new Error('Owner must be specified')
+    return {
+      doctype: this.doctype,
+      owners,
+    }
+  }
```

Comment:
We should make sure that the owner address is a CAIP-10 address: https://github.com/ceramicnetwork/specs/blob/master/doctypes/account-link.md#examples

Reply:
We can simply just assume ethereum mainnet for now (`<address>@eip155:1`)